src/cmd/reconcile.go

```diff
@@ -37,4 +40,5 @@ var reconcileCmd = &cobra.Command{
 func init() {
 	serviceCmd.AddCommand(reconcileCmd)
 	reconcileCmd.Flags().IntVar(&reconcileResyncInterval, "resync", 24, "The amount (in hours) before a full resync of the kubernetes cluster happens with OpsLevel.")
+	reconcileCmd.Flags().BoolVar(&disableServiceNameUpdate, "disable-service-name-update", true, "Turns off updating the service name.")
```

Comment:
I followed the existing flag setup of 
```
	rootCmd.PersistentFlags().Bool("disable-service-create", false, "Turns off automatic service creation (service data will still be reconciled). Overrides environment variable 'OPSLEVEL_DISABLE_SERVICE_CREATE'.")
```

But if this is confusing we could change it to be `--enable-service-name-update`

Reply:
🤔 Yes I think `--enable-service-name-update` would be clearer

Reply:
resolved